apps/full-site-editing/full-site-editing-plugin/blog-posts-block/newspack-homepage-articles/blocks/homepage-articles/view.php

```diff
@@ -123,15 +123,15 @@ function( $attribute ) {
 				if ( ! empty( $attributes['moreButtonText'] ) ) {
 					echo esc_html( $attributes['moreButtonText'] );
 				} else {
-					esc_html_e( 'Load more posts', 'newspack-blocks' );
+					esc_html_e( 'Load more posts', 'full-site-editing' );
 				}
 				?>
 				</button>
 				<p data-load-more-loading-text hidden>
-					<?php _e( 'Loading...', 'newspack-blocks' ); ?>
+					<?php _e( 'Loading...', 'full-site-editing' ); ?>
```

Comment:
Nit; `...` → `…`

Reply:
I think this will need to be done in the newspack source.